<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Guminis-Mac-mini.local" name="Default test" tests="2" failures="1" timestamp="4 May 2018 20:47:46 GMT" time="67.547" errors="0">
  <testcase name="methodOne" time="0.077" classname="com.lua.webbuyer.extent.reports.DummyForExReport"/>
  <testcase name="methodTwo" time="0.014" classname="com.lua.webbuyer.extent.reports.DummyForExReport">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
	at com.lua.webbuyer.extent.reports.DummyForExReport.methodTwo(DummyForExReport.java:47)
]]>
    </failure>
  </testcase> <!-- methodTwo -->
</testsuite> <!-- Default test -->
